src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '3.1.0'
+ModuleVersion = '3.2.0'
```

Comment:
@RandalliLama we will bump the versions during the release, so you can revert this change